### PR TITLE
[Dataproc module] Fix Variables

### PR DESCRIPTION
**Pull request by lcaggio**
In this PR we fix some variable bugs on the cluster configuration.

### CODE REVIEW

File: modules/dataproc/README.md [hunk 1]
```diff
@@ -46,6 +46,35 @@ module "processing-dp-cluster" {
 # tftest modules=1 resources=1
 ```
 
+### Cluster with CMEK encrypotion
```
Comment (ludoo) [inline modules/dataproc/README.md:49]: typo

File: modules/dataproc/README.md [hunk 2]
```diff
@@ -46,6 +46,35 @@ module "processing-dp-cluster" {
...
+
+To set cluster configuration use the Customer Managed Encryption key, set '' variable. The Compute Engine service agent and the Cloud Storage service agent needs to have 'CryptoKey Encrypter/Decrypter' role on they configured KMS key ([Documentation](https://cloud.google.com/dataproc/docs/concepts/configuring-clusters/customer-managed-encryption)).
```
Comment (ludoo) [inline modules/dataproc/README.md:51]: Lore, code quoting is via vackticks not single quotes. What's the meaning of the `set '' variable`?

Comment (ludoo) [reply modules/dataproc/README.md:51]: `and the Cloud Storage service agent needs` ---> `and the Cloud Storage service agent need`

**Review by ludoo — APPROVED** [review]

File: modules/dataproc/README.md (hunk 2 above)
Comment (lcaggio) [reply modules/dataproc/README.md]: Done

File: modules/dataproc/README.md (hunk 1 above)
Comment (lcaggio) [reply modules/dataproc/README.md]: Done

**Pull request merged**